slug.js

```diff
@@ -87,8 +87,9 @@
 
   function slug (string, opts) {
     let result = slugify(string, opts)
+    const fallback = opts?.fallback ?? slug.defaults.fallback
```

Comment:
This syntax doesn't work in Node.js 12. While Node.js 12 is EOL, dropping support for it would be a major version bump. I'm actually willing to do that, but if you'd prefer to keep this as a semver-minor bump, let's change the syntax here. What do you think?

Reply:
Sure, I haven't checked what version we aim for! I think that in the future it would be nice to extract options setup logic to its own function.